Makefile

```diff
@@ -16,5 +16,11 @@ build-test:
 	docker build -t openslides-auth-test -f Dockerfile.test .
 
 run-test: | build-test
+	echo "########################################################################"
+	echo "###################### Start full system tests #########################"
+	echo "########################################################################"
 	docker-compose -f docker-compose.test.yml up --abort-on-container-exit
+	docker-compose -f docker-compose.test.yml down
+
```

Comment:
Please add a `run-bash`: This should start up the contianers and run `docker-compose -f docker-compose.test.yml exec test bash` -> This is for quicker development. In the given shell, you can run `npm run test` over and over agian wthout starting and stopping everything.